lib/driver.js

```diff
@@ -100,6 +100,7 @@ class XCUITestDriver extends BaseDriver {
       let { device, udid, realDevice } = await this.determineDevice();
       log.info(`Determining device to run tests on: udid: '${udid}', real device: ${realDevice}`);
       this.device = device;
```

Comment:
No, one place is better. I just wanted there to be a reason. I'm good with `opts.device`.